src/services/users.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+let users = [];
+
+function getUsers() {
+  users = [];
+
+  return users;
+}
+
+function getAll() {
+  return users;
+}
+
+function getById(userId) {
+  const getUsersById = users.find(user => user.id === parseInt(userId));
+
+  return getUsersById || null;
+}
+
+function create(name) {
+  const maxId = Math.max(users.map(user => user.id), 0);
```

Comment:
The Math.max function is not applied correctly. It should be Math.max(...users.map(user => user.id), 0) to spread the id array.